sw/device/lib/dif/dif_aes_unittest.cc

```diff
@@ -42,15 +42,43 @@ class AesTest : public testing::Test, public mock_mmio::MmioTest {
       EXPECT_WRITE32(offset, iv.iv[i]);
     }
   }
-  void SetExpectedConfig(uint32_t key_len, uint32_t mode, uint32_t operation,
-                         bool manual_op, bool force_zero_masks) {
+
+  struct ConfigOptions {
+    uint32_t key_len;
```

Comment:
You need to make sure you provide default values such as `=0` or `=false`; otherwise you can wind up with nasty surprises.

Reply:
+1 to @mcy's comment. It is best practice to add default initializers to struct members in C++. This applies to the other structs too.

Reply:
Done